feature/booth/src/main/kotlin/com/unifest/android/feature/booth/component/BoothItem.kt

```diff
@@ -37,24 +39,26 @@ internal fun BoothItem(
     booth: BoothTabModel,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        modifier = modifier,
+    Row(
+        modifier = modifier
+            .height(86.dp),
```

Comment:
속성이 하나만 설정된 경우엔 한줄로 표기하는게 컨벤션이긴 했어서ㅎ 고렇게 표기해주시면 좋을거같슴다
```suggestion
modifier = modifier.height(86.dp),
```